ztp/gitops-subscriptions/argocd/deployment/resource-hook-role-binding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: converter
+  namespace: sites-sub 
```

Comment:
namespace must be clusters-sub. sites-sub used by sites-app which is for site specific policies.

Reply:
fixed